### ISSUE TITLE
Old navbar, grade report and floatingheaders.js

### ISSUE THREAD

**Issue by gjb2048** [gjb2048/moodle-theme_essential#457]
M2.8+ issue....

floatingheaders.js sometimes gets the calculation right / sometimes wrong.  Depends on how far down the page you scroll.  Clean appears to be fine.

Using 'oldnavbar' set.  Thus any theme where the navbar is not at the exact top could suffer.

![2015-02-22 15_26_59-grades_ view](https://cloud.githubusercontent.com/assets/1058419/6319085/e022a208-baa7-11e4-8cdc-7e2fffd07994.jpg)

**Comment by gjb2048** [comment]
![2015-02-22 15_55_25-f__moodledev_moodle28_grade_report_grader_yui_src_gradereporttable_js_floatinghe](https://cloud.githubusercontent.com/assets/1058419/6319205/718a9680-baab-11e4-808a-8f23f7d2e392.jpg)

**Comment by gjb2048** [comment]
I believe that I need help with this, that potentially, core code needs to evolve in order to make it possible for a contributed theme such as this to adapt.

**Comment by gjb2048** [comment]
Is this still an issue in M3.0?

**Comment by gjb2048** [comment]
Ref: https://moodle.org/mod/forum/discuss.php?d=327579#p1319145

Possible, but would need verifying.

````
.nav-pills, .nav-tabs {
    text-align: left !important;
}

.path-grade-report-grader .gradeparent {
  position: relative;
}

#page-grade-report-grader-index #page #page-content #region-main {
  overflow-x: visible;
  width: auto;
}

.path-grade-report-grader .gradeparent {
  position: relative;
}

#page-grade-report-grader-index #page #page-content #region-main {
  overflow-x: visible;
  width: auto;
}

#page {
  overflow: visible !important;
}
````

**Comment by gjb2048** [comment]
Fixed in #641.
